src/state.rs

```diff
@@ -74,9 +77,25 @@ impl VMState {
             read_storage_slots: RollbackableHashSet::<StorageKey>::default(),
             written_storage_slots: RollbackableHashSet::<StorageKey>::default(),
             decommitted_hashes: RollbackableHashSet::<U256>::default(),
+            initial_values: HashMap::default(),
         }
     }
 
+    pub fn reset(&mut self) {
```

Comment:
If we're setting everything as default, we can simply do Default::default() here.